apps/aeoracle/src/aeo_response_tx.erl

```diff
@@ -134,9 +134,9 @@ check(#oracle_response_tx{nonce = Nonce, query_id = QueryId,
                      aetx_contract -> []; %% TODO: Handle fees from contracts right.
                      aetx_transaction ->
                          [fun() -> aetx_utils:check_account(OraclePubKey, Trees,
-                                                            Nonce, Fee - QueryFee) end,
-                          fun() -> aeo_utils:check_ttl_fee(Height, ResponseTTL, Fee) end]
+                                                            Nonce, Fee - QueryFee) end]
```

Comment:
This will be impacted by https://github.com/aeternity/epoch/pull/1792.

Reply:
I see this is now rebased on 1792. Ok.